source/agora/test/Base.d

```diff
@@ -179,6 +180,9 @@ private UnitTestResult customModuleUnitTester ()
     foreach (mod; parallel(mod_tests))
         runTest(mod);
 
+    foreach (mod; parallel(heavy_tests))
+        runTest(mod);
```

Comment:
Should they be run in parallel ?

Reply:
Well there's only one such test, but I'm thinking there might be more in the future.